### DISCUSSION TITLE
Theme color key for multi line selection

### DISCUSSION THREAD

**Discussion by Proful** [helix-editor/helix#4985] [Q&A]
I want to use one color for the 3 line selected. Below selections looks more cluttered.
Can I customise via a theme key?

![image](https://user-images.githubusercontent.com/354596/205434913-f4605d31-cd65-4166-aaae-b6c3cb2b1f4c.png)

**Comment by the-mikedavis** [comment]
The `ui.selection` key is used to theme selections (note that the cursor is themed by `ui.cursor` though). Also see https://docs.helix-editor.com/master/themes.html

**Comment by Proful** [comment]
Thanks.

Is it possible to change style of selection of entire line vs single word (two different style).

I'm happy with default selection of a word. I don't want to change that style.
```"ui.selection" = { modifiers = [ "reversed"] }```

However, when selecting an entire line or multiple line, I want one single background.

**Reply by the-mikedavis** [reply]
> Is it possible to change style of selection of entire line vs single word

No this is not currently possible
